medusa/helpers/__init__.py

```diff
@@ -1728,7 +1728,13 @@ def title_to_imdb(title, start_year, imdb_api=None):
     if imdb_api is None:
         imdb_api = Imdb()
 
-    titles = imdb_api.search_for_title(title)
+    try:
+        titles = imdb_api.search_for_title(title)
+    except ValueError as error:
+        # FIXME: Putting an error here, as this is a known error with the lib imdbpie. And should be fix.
+        log.error('Could not get a result from imdbpie for the title {title}, error: {error}',
```

Comment:
Since we know we need to fix it upstream, shouldn't it be debug? We likely don't want users to report it, right?